posts/2020-09-10-survey-launch.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: post
+title: "Launching the 2020 State of Rust Survey"
+author: The Rust Community Team
+description: "Hearing from you about the fifth year of Rust"
+---
+
+It's that time again! Time for us to take a look at how the Rust project is doing, and what we should plan for the future. The Rust Community Team is pleased to announce our [2020 State of Rust Survey][survey]! Whether or not you use Rust today, we want to know your opinions. Your responses will help the project understand its strengths and weaknesses and establish development priorities for the future. (If you'd like to give longer form feedback on the Rust roadmap, [we're also collecting blog posts!])
```

Comment:
> [we're also collecting blog posts!]

I think the intent was to have a link either here: 
https://blog.rust-lang.org/2020/09/03/Planning-2021-Roadmap.html

or here
https://forms.gle/Hv41uA5qJEY89XRm7